astropy/units/tests/test_quantity_erfa_ufuncs.py

```diff
@@ -504,3 +509,93 @@ def test_apio(self):
                 rtol=1e-12,
                 atol=0 * self.astrom_unit[name],
             )
+
+
+class TestGeodetic:
+    def setup_class(self):
+        self.ellipsoid = 1
+        self.length_unit = u.Unit("m")
+        self.equatorial_radius_value = 6378136.0
+        self.equatorial_radius = self.equatorial_radius_value << self.length_unit
+        self.flattening = 0.0033528 * u.dimensionless_unscaled
+        self.lon_value = 0.9827937232473290680
+        self.lon_unit = u.Unit("rad")
+        self.lon = self.lon_value << self.lon_unit
+        self.lat_value = 0.9716018377570411532
+        self.lat_unit = u.Unit("rad")
+        self.lat = self.lat_value << self.lat_unit
+        self.height_value = 332.36862495764397
+        self.height = self.height_value << self.length_unit
+        self.x_value = 2e6
+        self.x = self.x_value << self.length_unit
+        self.y_value = 3e6
+        self.y = self.y_value << self.length_unit
+        self.z_value = 5.244e6
+        self.z = self.z_value << self.length_unit
+
+    def test_unit_errors(self):
+        """Test unit errors when dimensionless parameters are used"""
+
+        msg = "'NoneType' object has no attribute '_get_converter'"
+        xyz = np.array([self.x_value, self.y_value, self.z_value]) << self.length_unit
+        status = 0
```

Comment:
No need to set `status = 0` beforehand, since you're not passing it in. Indeed, since we're just looking for the error message, you can just call the `ufunc` without assigning to any outputs.